pkgs/development/python-modules/energyflow/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, h5py
+, numpy
+, six
+, wasserstein
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "EnergyFlow";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "pkomiske";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-fjT8c0ZTjdufP334upPzRVdTJDIBs84I7PkFu4CMcQw=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.cfg \
+      --replace "setup_requires=" "" \
+      --replace "pytest-runner" ""
+  '';
+
+  propagatedBuildInputs = [
+    h5py
+    numpy
+    six
+    wasserstein
+  ];
+
+  doCheck = true;
+  checkInputs = [
+    pytestCheckHook
+  ];
+  pytestFlagsArray = [
+    "energyflow/tests"
+  ];
+  preCheck = ''
+    rm energyflow/tests/test_archs.py # requires tensorflow
+    rm energyflow/tests/test_emd.py # requires "ot"
+  '';
```

We do have tensorflow, but I assume you don't want to create a dependency onto that?

```suggestion
  disabledTestPaths = ['
    "energyflow/tests/test_archs.py" # requires tensorflow
    "energyflow/tests/test_emd.py" # requires "ot
  ];
```

---

Seems like people are not so careful when it comes to tensorflow maintenance in nixpkgs. I'd skip on this dependency for now.